bitnami/chainloop/templates/NOTES.txt

```diff
@@ -67,4 +67,4 @@ https://docs.chainloop.dev/getting-started/installation#configure-cli-optional
 {{- include "common.warnings.rollingTag" .Values.controlplane.image }}
 {{- include "common.warnings.rollingTag" .Values.cas.image }}
 {{- include "common.warnings.rollingTag" .Values.controlplane.migration.image }}
-{{- include "common.warnings.modifiedImages" (dict "images" (list .Values.controlplane.image .Values.cas.image .Values.controlplane.migration.image) "context" $) }}
+{{- include "common.warnings.modifiedImages" (dict "images" (list .Values.controlplane.image .Values.cas.image .Values.controlplane.migration.image .Values.dex.image) "context" $) }}
```

Comment:
Dex is a valid image, isn't it?
https://github.com/bitnami/charts/blob/788bb4a7ba51e14a1a08abac4300332b21688936/bitnami/chainloop/values.yaml#L1744